packages/contracts-bedrock/deploy-config/mainnet.json

```diff
@@ -28,8 +28,8 @@
   "l2GenesisBlockGasLimit": "0x1c9c380",
   "l2GenesisBlockCoinbase": "0x4200000000000000000000000000000000000011",
   "l2GenesisBlockBaseFeePerGas": "0x3b9aca00",
-  "gasPriceOracleOverhead": 2100,
-  "gasPriceOracleScalar": 1000000,
+  "gasPriceOracleOverhead": 188,
+  "gasPriceOracleScalar": 684000,
   "eip1559Denominator": 50,
   "eip1559Elasticity": 10,
```

Comment:
Should be 6: see https://www.notion.so/oplabs/Optimism-Gas-Target-1559-Params-for-Mainnet-Launch-864e8f41e68f4743b74ac029b67fbeef